src/MapTrack3D.tsx

```diff
@@ -39,26 +49,37 @@ export const MapTrack3D: React.FC<Props> = ({ options, data, width, height }) =>
     camera: null,
     renderer: null,
     animationRequestId: null,
+    texturePath: options.customTextureURL === '' ? earthTexture : options.customTextureURL,
```

Comment:
In the case of the texturePath, I think it would be better off as useMemo. This is because it is not important that we store one specific string, rather that the value does not change.